## Confidence Interval 

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

In [3]:
list=[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]
list

[1.13,
 1.55,
 1.43,
 0.92,
 1.25,
 1.36,
 1.32,
 0.85,
 1.07,
 1.48,
 1.2,
 1.33,
 1.18,
 1.22,
 1.29]

## Converted the list data in to DataFrame using below code.

In [9]:
df=pd.DataFrame({'Print_Life':[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]})
df


Print_Life
0         1.13
1         1.55
2         1.43
3         0.92
4         1.25
5         1.36
6         1.32
7         0.85
8         1.07
9         1.48
10        1.20
11        1.33
12        1.18
13        1.22
14        1.29

## Using the Pandas function describe found out the mean and standard deviation for the sample.

In [11]:
df.describe()

Print_Life
count   15.000000
mean     1.238667
std      0.193164
min      0.850000
25%      1.155000
50%      1.250000
75%      1.345000
max      1.550000

In [13]:
n=15
X_bar=1.238667
Alpha=0.01
sample_std=0.193164

## With the help of below code calculated the minimum and maximum lifetime of the printing head with 99% confidence.

In [20]:
stats.t.interval(0.99, df = 14, loc = X_bar, scale = (sample_std/np.sqrt(15)))

(1.090197771361284, 1.3871362286387159)

## With 99% confidence we can say that the minimum lifetime of the printing tool is 1.09 and the maximum lifetime is 1.39.